api/api/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.serializers import ModelSerializer
+from api.models import Department, Discipline, Class
+
+class DepartmentSerializer(ModelSerializer):
+    class Meta:
+        model = Department
+        fields = ('code', 'year', 'period')
```

Comment:
Observa-se na [documentação](https://www.django-rest-framework.org/api-guide/serializers/#specifying-which-fields-to-include) do Django Rest Framework que caso você queira que o `ModelSerializer` realize as operações com _todos_ os campos da minha `model` que foi passada não é necessário lista-los, apenas colocar `'__all__'`. O mesmo segue para as linhas 12 e 20. Recomendo rodar os testes novamente após a alteração par ter certeza da troca.